packages/client/src/components/ProductList/ProductList.component.js

```diff
@@ -25,6 +25,8 @@ export default function ProductList({ isLoading, products, filteredProducts }) {
 }
 ProductList.propTypes = {
   isLoading: PropTypes.bool.isRequired,
-  products: PropTypes.arrayOf(PropTypes.string).isRequired,
-  filteredProducts: PropTypes.arrayOf(PropTypes.string).isRequired,
+  products: PropTypes.arrayOf(PropTypes.oneOfType([PropTypes.object]))
```

Comment:
Here, it would be much better if you used `PropTypes.shape`, like:
```
PropTypes.arrayOf(PropTypes.shape({
  id: PropTypes.number,
  title: PropTypes.string,
})).isRequired
```

Now, you'll likely use the shape for Products in many components since it's a core domain object, so best to create it once and re-use it, like:
```
export const ProductPropType = PropTypes.shape({
  id: PropTypes.number,
  title: PropTypes.string,
});

// Other file
import { ProductPropType } from './prop-types';

PropTypes.arrayOf(ProductPropType).isRequired
```

This also makes the code much more clear!